src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_help.py

```diff
@@ -72,14 +77,34 @@
     short-summary: commands to configure hostnames
 """
 
+helps['appservice web deployment'] = """
+    type: group
+    short-summary: Commands to manage web app deployments
```

Comment:
Agree on consistency. I kept "Commands to..." for groups. Also just a preference, I could be convinced to go a different direction, but I kind of like the way it reads for a group - it helps clarify that it's not a single command.